ui/package.json

```diff
@@ -12,7 +12,8 @@
     "lint": "./node_modules/@angular/cli/bin/ng.js lint",
     "e2e": "./node_modules/@angular/cli/bin/ng.js e2e",
     "ci-test": "./node_modules/@angular/cli/bin/ng.js test --no-watch --no-progress --karma-config=karma.ci.conf.js",
-    "clean": "rm -rf ../api/src/main/resources/ui/*; rm -rf coverage; rm -rf reports; rm -rf test-results; rm -rf node_modules"
+    "clean": "rm -rf ../api/src/main/resources/ui/*; rm -rf coverage; rm -rf reports; rm -rf test-results; rm -rf node_modules",
+    "wdio": "wdio run ./wdio.conf.ts"
```

Comment:
What's the difference between this and line 13? Should we have both? I would not know to look for "wdio" as an e2e suite, so what is a typical convention here?

Reply:
You're correct. This is run using npm run e2e which leverages the `angular.json -> architect -> e2e`